config/samples/v2/external-object-storage/dspa.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: datasciencepipelinesapplications.opendatahub.io/v1alpha1
```

Comment:
can you reduce this down to the new v2 simple: https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/samples/dspa_simple_v2.yaml

Reply:
Done.